media-plugins/distrho-ports/distrho-ports-9999.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3
+
+DESCRIPTION="Linux ports of Distrho plugins."
+HOMEPAGE="http://distrho.sourceforge.net"
+
+EGIT_BRANCH="master"
+EGIT_REPO_URI="https://github.com/DISTRHO/DISTRHO-Ports.git"
+
+LICENSE="GPL-2"
+SLOT="0"
+
+RDEPEND="media-libs/alsa-lib
```

Comment:
Seems like you're missing some deps here as @marcan mentioned https://github.com/gentoo-audio/audio-overlay/pull/50#discussion_r124906114

These deps should all be defined. In case you're not sure I find it helps to search through the ebuilds in the gentoo tree for how it's done there:
- https://github.com/gentoo/gentoo/search?l=Gentoo+Ebuild&q=freetype&type=&utf8=%E2%9C%93
- https://github.com/gentoo/gentoo/search?l=Gentoo+Ebuild&q=xrender&type=&utf8=%E2%9C%93
- https://github.com/gentoo/gentoo/search?l=Gentoo+Ebuild&q=xcursor&type=&utf8=%E2%9C%93

As @marcan said if they're optional they should be behind a USE flag to avoid automagic dependency usage.